azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/handlers/runtime/DockerFunctionRuntimeHandler.java

```diff
@@ -18,10 +18,10 @@
 import static com.microsoft.azure.maven.Utils.assureServerExist;
 import static com.microsoft.azure.maven.appservice.DockerImageType.PUBLIC_DOCKER_HUB;
 import static com.microsoft.azure.maven.function.Constants.APP_SETTING_FUNCTION_APP_EDIT_MODE;
-import static com.microsoft.azure.maven.function.Constants.APP_SETTING_FUNCTION_APP_EDIT_MODE_VALUE;
+import static com.microsoft.azure.maven.function.Constants.APP_SETTING_FUNCTION_APP_EDIT_MODE_READONLY;
```

Comment:
use star for many constants

Reply:
Will fix it in later PR

Reply:
We enable the `AvoidStarImport` in checkstlye, which will block star imports.